<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassWordChange</class>
 <widget class="QDialog" name="PassWordChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密码修改</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>2</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font-size:25px;
font: bold;</string>
           </property>
           <property name="text">
            <string>修改密码</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_13">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>3</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="labelAlignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">font-size:18px;</string>
             </property>
             <property name="text">
              <string>旧密码：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="oldPassword">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    color: #1e222b;
    padding: 1px 3px;  /* 去掉右边的15px padding，因为LineEdit不需要下拉按钮的空间 */
    border: 1px solid rgba(228,228,228,1);
    border-radius: 8px;
    font-family: Microsoft YaHei;
    font-size: 14px;
    background: rgba(255,255,255,1);
}

QLineEdit:focus {
    border: 1px solid rgba(228,228,228,1);  /* 保持聚焦时边框样式一致 */
    outline: none;
}

/* 垂直滚动条样式，如果LineEdit是多行的话 */
QLineEdit QScrollBar:vertical {
    width: 6px;
    background: rgb(255,255,255);
    border: none;
}

QLineEdit QScrollBar::handle:vertical {
    border-radius: 3px;
    background-color: rgba(30, 34, 43, 0.2);
}

QLineEdit QScrollBar::handle:vertical:hover {
    background-color: rgba(30, 34, 43, 0.2);
}

QLineEdit QScrollBar::add-line:vertical {
    border: none;
}

QLineEdit QScrollBar::sub-line:vertical {
    border: none;
}</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>请输入旧密码</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font-size:18px;</string>
             </property>
             <property name="text">
              <string>新密码：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="passwordEdit">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    color: #1e222b;
    padding: 1px 3px;  /* 去掉右边的15px padding，因为LineEdit不需要下拉按钮的空间 */
    border: 1px solid rgba(228,228,228,1);
    border-radius: 8px;
    font-family: Microsoft YaHei;
    font-size: 14px;
    background: rgba(255,255,255,1);
}

QLineEdit:focus {
    border: 1px solid rgba(228,228,228,1);  /* 保持聚焦时边框样式一致 */
    outline: none;
}

/* 垂直滚动条样式，如果LineEdit是多行的话 */
QLineEdit QScrollBar:vertical {
    width: 6px;
    background: rgb(255,255,255);
    border: none;
}

QLineEdit QScrollBar::handle:vertical {
    border-radius: 3px;
    background-color: rgba(30, 34, 43, 0.2);
}

QLineEdit QScrollBar::handle:vertical:hover {
    background-color: rgba(30, 34, 43, 0.2);
}

QLineEdit QScrollBar::add-line:vertical {
    border: none;
}

QLineEdit QScrollBar::sub-line:vertical {
    border: none;
}</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::EchoMode::Password</enum>
             </property>
             <property name="placeholderText">
              <string>请输入新密码</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">font-size:18px;</string>
             </property>
             <property name="text">
              <string>确认密码：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="passwordConfirm">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    color: #1e222b;
    padding: 1px 3px;  /* 去掉右边的15px padding，因为LineEdit不需要下拉按钮的空间 */
    border: 1px solid rgba(228,228,228,1);
    border-radius: 8px;
    font-family: Microsoft YaHei;
    font-size: 14px;
    background: rgba(255,255,255,1);
}

QLineEdit:focus {
    border: 1px solid rgba(228,228,228,1);  /* 保持聚焦时边框样式一致 */
    outline: none;
}

/* 垂直滚动条样式，如果LineEdit是多行的话 */
QLineEdit QScrollBar:vertical {
    width: 6px;
    background: rgb(255,255,255);
    border: none;
}

QLineEdit QScrollBar::handle:vertical {
    border-radius: 3px;
    background-color: rgba(30, 34, 43, 0.2);
}

QLineEdit QScrollBar::handle:vertical:hover {
    background-color: rgba(30, 34, 43, 0.2);
}

QLineEdit QScrollBar::add-line:vertical {
    border: none;
}

QLineEdit QScrollBar::sub-line:vertical {
    border: none;
}</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::EchoMode::Password</enum>
             </property>
             <property name="placeholderText">
              <string>请确认新密码</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    color: #1e222b;
    background-color: #ffffff;
    border: 1px solid rgba(228,228,228,1);
    border-radius: 8px;
    font-family: Microsoft YaHei;
    font-size: 14px;
    padding: 1px 15px;
    min-height: 24px;
    outline: none;  /* 添加这行去除焦点边框 */
}

QPushButton:focus {  /* 添加focus状态样式 */
    outline: none;
    border: 1px solid rgba(228,228,228,1);
}

QPushButton:hover {
    background-color: rgba(30, 34, 43, 0.05);
    border: 1px solid rgba(30, 34, 43, 0.2);
}

QPushButton:pressed {
    background-color: rgba(30, 34, 43, 0.1);
    border: 1px solid rgba(30, 34, 43, 0.3);
    padding: 2px 15px 0px 15px;
}

QPushButton:disabled {
    color: rgba(30, 34, 43, 0.3);
    background-color: rgba(228,228,228,0.3);
    border: 1px solid rgba(228,228,228,0.5);
}

QPushButton:icon {
    padding: 1px 15px 1px 10px;
}</string>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
           </property>
           <property name="centerButtons">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_10">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>19</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
